docs/DeveloperGuide.md

```diff
@@ -36,3 +233,32 @@
 ## Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+## Command Summary
+
```

Comment:
The visuals are well integrated to the explanation. The actions-to-perform and the format are shown clearly.